includes/woopay/class-woopay-session.php

```diff
@@ -31,7 +31,7 @@
  */
 class WooPay_Session {
 
-	const STORE_API_NAMESPACE_PATTERN = '@^wc/store(/v[\d]+)?$@';
+	const STORE_API_NAMESPACE_PATTERN = '@^(wc/store(/v[\d]+)?|store-api)$@';
```

Comment:
The Store API changed its "issuer" value: https://github.com/woocommerce/woocommerce/blob/1a5c8d69e8447fc1a0cff1333b1bdaa8a419f218/plugins/woocommerce/src/StoreApi/Utilities/CartTokenUtils.php#L23-L32